### PR TITLE
Update/0.5.3

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 0.5.3 (py 3.11 compat)

https://github.com/pydata/patsy/releases

### CODE REVIEW

**Review by marcoesters — APPROVED** [review]
You can remove `doc_source_url` - LGTM otherwise

**Comment by cbouss** [comment]
> You can remove `doc_source_url` - LGTM otherwise

Done, thank you

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,10 +8,10 @@ package:
...
 source:
   url: https://pypi.io/packages/source/{{ pkg_name[0] }}/{{ pkg_name }}/{{ pkg_name }}-{{ version }}.tar.gz
-  sha256: 5053de7804676aba62783dbb0f23a2b3d74e35e5bfa238b88b7cbf148a38b69d
+  sha256: bdc18001875e319bc91c812c1eb6a10be4bb13cb81eb763f466179dca3b67277
 
 build:
-  number: 1
+  number: 0
   script:
     - python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: Can be updated to `{{ PYTHON }} -m pip install . --no-deps -vv`

Comment (cbouss) [reply recipe/meta.yaml]: thank you, addressed in commit.

**Pull request merged**